src/services/ipfs/ipfs.resolver.ts

```diff
@@ -0,0 +1,34 @@
+import { IpfsUploadFailedException } from '@common/exceptions/ipfs.exception';
+import { Inject } from '@nestjs/common';
+import { Args, Mutation, Resolver } from '@nestjs/graphql';
+import { createWriteStream } from 'fs';
+import { FileUpload, GraphQLUpload } from 'graphql-upload';
+import { IpfsService } from './ipfs.service';
+import { v4 as uuidv4 } from 'uuid';
+
+@Resolver()
+export class IpfsResolver {
+  constructor(@Inject(IpfsService) private ipfsService: IpfsService) {}
+
+  @Mutation(() => String)
```

Comment:
user should be registered? do not want this world open.

Reply:
Addressed in 25eb4f5a65de0e22a10bcddbc7317ac9e2360584